packages/react/src/components/List/ListItem/ListItem.jsx

```diff
@@ -39,7 +39,14 @@ const ListItemPropTypes = {
   expanded: PropTypes.bool,
   value: PropTypes.string.isRequired,
   /** string value or callback render function */
-  secondaryValue: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
+  secondaryValue: PropTypes.oneOfType([
```

Comment:
We should also update the documentation in packages/react/src/components/List/List.mdx